modules/cli/src/main/scala/scala/cli/commands/Uninstall.scala

```diff
@@ -0,0 +1,49 @@
+package scala.cli.commands
+
+import caseapp._
+
+import scala.cli.CurrentParams
+import scala.cli.commands.util.VerbosityOptionsUtil._
+
+object Uninstall extends ScalaCommand[UninstallOptions] {
+  def run(options: UninstallOptions, args: RemainingArgs): Unit = {
+    CurrentParams.verbosity = options.bloopExit.logging.verbosityOptions.verbosity
+    val interactive =
+      options.bloopExit.logging.verbosityOptions.interactiveInstance(forceEnable = true)
+
+    val binDirPath =
+      options.binDirPath.getOrElse(scala.build.Directories.default().binRepoDir / "scala-cli")
+    val destBinPath = binDirPath / options.binaryName
+    val cacheDir    = scala.build.Directories.default().cacheDir
+
+    if (
+      !Update.isScalaCLIInstalledByInstallationScript() && (options.binDir == None || (options.binDir != None && !options.force))
+    ) {
+      System.err.println(
+        "Scala CLI was not installed by the installation script, please use your package manager to uninstall scala-cli."
+      )
+      sys.exit(1)
+    }
+    if (!options.force) {
+      val fallbackAction = () => {
+        System.err.println(s"To uninstall scala-cli pass -f or --force")
+        sys.exit(1)
+      }
+      val msg = s"Do you want to uninstall scala-cli?"
+      interactive.confirmOperation(msg).getOrElse(fallbackAction())
+    }
+    if (os.exists(destBinPath)) {
+      // exit bloop server
+      BloopExit.run(options.bloopExit, args)
+      // remove scala-cli launcher
+      os.remove.all(binDirPath)
+      // remove scala-cli caches
+      if (!options.skipCache) os.remove.all(cacheDir)
+      println("Uninstalled sucessfully")
```

Comment:
NIT Better use a logger here and call `logger.message` here. That way, the message won't be printed in quiet mode;
```suggestion
      logger.message("Uninstalled sucessfully")
```
(needs `logger` to be created above, with `options.bloopExit.logging.logger` I think).